docs/tutorials/adapters.md

```diff
@@ -3,48 +3,47 @@
 OpenTimelineIO supports, or plans to support, conversion adapters for many
 existing file formats.
 
-### Final Cut Pro XML ###
+## Final Cut Pro XML
 
 Final Cut 7 XML Format
 - Status: Supported via the `fcp_xml` adapter
-- <a href="https://developer.apple.com/library/content/documentation/AppleApplications/Reference/FinalCutPro_XML/AboutThisDoc/AboutThisDoc.html#//apple_ref/doc/uid/TP30001152-TPXREF101" target="_blank">Reference</a>
+- [Reference](https://developer.apple.com/library/archive/documentation/AppleApplications/Reference/FinalCutPro_XML/AboutThisDoc/AboutThisDoc.html#//apple_ref/doc/uid/TP30001152-TPXREF101)
 
 Final Cut Pro X XML Format:
 - Status: Supported via the `fcpx_xml` adapter
-- <a href="https://developer.apple.com/library/mac/documentation/FinalCutProX/Reference/FinalCutProXXMLFormat/Introduction/Introduction.html" target="_blank">Intro to FCP X XML</a>
+- [Intro to FCP X XML](https://developer.apple.com/library/mac/documentation/FinalCutProX/Reference/FinalCutProXXMLFormat/Introduction/Introduction.html)
 
-### Adobe Premiere Project ###
+## Adobe Premiere Project
 
 - Based on guidance from Adobe, we support interchange with Adobe Premiere via
     the FCP 7 XML format (see above).
 
-### CMX3600 EDL ###
+## CMX3600 EDL
 
 - Status: Supported via the `cmx_3600` adapter
 - Includes support for ASC_CDL color correction metadata
 - Full specification: SMPTE 258M-2004 "For Television −− Transfer of Edit Decision Lists"
 - http://xmil.biz/EDL-X/CMX3600.pdf
-- <a href="https://documentation.apple.com/en/finalcutpro/usermanual/index.html#chapter=96%26section=1" target="_blank">Reference</a>
+- [Reference](https://prohelp.apple.com/finalcutpro_help-r01/English/en/finalcutpro/usermanual/chapter_96_section_0.html)
```

Comment:
This link is now fixed. It was dead before and @jminor suggested an internet archive link. But after some searching, I found this link that is "exactly" what was on the internet archive. I used the section 0 instead of 1, but that's just because of the documentation layout that changed slightly. The content is still the same as far as I see.